files/image_config/hostcfgd/hostcfgd

```diff
@@ -43,6 +42,139 @@ def obfuscate(data):
         return data
 
 
+class Feature(object):
```

Comment:
can you add unit test for this?

Reply:
I think it can be added after https://github.com/Azure/sonic-buildimage/pull/5713 which moves hostcfgd to a python package with tests and I can write test in https://github.com/Azure/sonic-buildimage/tree/f4083627f163863f1c2b9e4038c382e3941db218/src/sonic-host-services/tests

Reply:
Done